website/Contribution/CONTRIBUTING.rst

```diff
@@ -884,15 +884,16 @@ Models
 
 * In a Model attribute order should be
 
-  1. Private attributes (`_name`, `_description`, `_inherit`, ...)
-  2. Fields declarations
-  3. Default method and `_default_get`
-  4. Compute and search methods in the same order than field declaration
-  5. Constrains methods (`@api.constrains`) and onchange methods
+  #. Private attributes (`_name`, `_description`, `_inherit`, ...)
+  #. Fields declarations
+  #. SQL constraints
```

Comment:
Why not in second place?

Reply:
Almost all classes I've seen have SQL constraint declared after fields. In addition, personally, I think it makes more sense for readability, otherwise you would read constraints involving fields not declared yet.